app/src/main/java/fr/free/nrw/commons/feedback/FeedbackContentCreator.java

```diff
@@ -0,0 +1,77 @@
+package fr.free.nrw.commons.feedback;
+
+import android.content.Context;
+import fr.free.nrw.commons.R;
+import fr.free.nrw.commons.feedback.model.Feedback;
+import fr.free.nrw.commons.utils.LangCodeUtils;
+import java.util.Locale;
+
+/**
+ * Creates a wikimedia recognizable format
+ * from feedback information
+ */
+public class FeedbackContentCreator {
+    private StringBuilder stringBuilder;
+    private Feedback feedback;
+    private Context context;
+
+    public FeedbackContentCreator(Context context, Feedback feedback) {
+        this.feedback = feedback;
+        this.context = context;
+        init();
+    }
+
+    /**
+     * Initializes the string buffer object to append content from feedback object
+     */
+    public void init() {
+        // Localization is not needed here, because this ends up on a page where developers read the feedback, so English is the most convenient.
+
+        stringBuilder = new StringBuilder();
+        stringBuilder.append("== ").append(LangCodeUtils.getLocalizedResources(context,
+            Locale.ENGLISH).getString(R.string.feedback_header)).append(feedback.getVersion()).append(" ==");
+        stringBuilder.append("\n");
+        stringBuilder.append(feedback.getTitle());
+        stringBuilder.append("\n");
+        stringBuilder.append("\n");
+        if (feedback.getApiLevel() != null) {
+            stringBuilder.append("* ").append(LangCodeUtils.getLocalizedResources(context,
```

Comment:
Similarly, how about doing like this for readability?
```
stringBuilder.append("* ");
stringBuilder.append(
  LangCodeUtils.getLocalizedResources(context, Locale.ENGLISH)
    .getString(R.string.api_level));
stringBuilder.append(": ");
stringBuilder.append(feedback.getApiLevel());
stringBuilder.append("\n");
```
Same for the blocks below.